docs/cassandra/multi-az.md

```diff
@@ -0,0 +1,51 @@
+Cassandra Across Multiple Availability Zones
+============================================
+
+Navigator supports running Cassandra with
+[rack and datacenter-aware replication](https://docs.datastax.com/en/cassandra/latest/cassandra/architecture/archDataDistributeReplication.html).
+To deploy this, you must run a `nodePool` in each availability zone, and mark each as a separate Cassandra rack.
+
+The `nodeSelector` field of a nodePool allows scheduling the nodePool to a set of nodes matching labels.
+This should be used with a node label such as
+[`failure-domain.beta.kubernetes.io/zone`](https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesiozone).
```

Comment:
Would be good to add a link out to the official docs on nodeSelector. It's not a concept we've invented and there are probably better descriptions out there that we want to include :smile:

Reply:
:+1: